### PR TITLE
DCS-1239 Initial commit of v2 set of additional conditions

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: server/routes/config/additionalConditionsV1.ts [hunk 1]
```diff
@@ -0,0 +1,332 @@
+export const additionalConditionsV1 = {
```
Comment (andrewrlee) [inline server/routes/config/additionalConditionsV1.ts:1]: extracted out separate modules for validation messages for each set of conditions

File: server/routes/config/licenceConditions.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      },
+    },
+  ],
+  modificationRequiresApproval: true,
+}
+
+export const additional = new Map<ConditionVersion, any>([
+  [1, additionalConditionsV1],
+  [2, additionalConditionsV2],
+])
```
Comment (andrewrlee) [inline server/routes/config/licenceConditions.ts:20]: move to typescript and turned this from an array of the original single set of conditions to a map to allow retrieving conditions by version

File: server/services/utils/formValidation.ts [hunk 3]
```diff
@@ -147,7 +143,16 @@ const validationProcedures = {
     getErrorMessage: (fieldConfig, errorPath) => getIn(fieldConfig, [...errorPath, 'validationMessage']),
   },
   additional: {
-    getSchema: () => additionalConditionsSchema,
+    getSchema: ({ version }) => {
```
Comment (andrewrlee) [inline server/services/utils/formValidation.ts:146]: There's at least 3 representations of additional config:
* Form config
* Validation messages
* Validation schema

Here it reads the version from the form config to work out which schema to use.

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**